_videos/2017-01-30-complex-refactoring-in-simple-steps-III.md

```diff
@@ -0,0 +1,21 @@
+---
+author: Matthew Butt
+layout: video
+name: complex-refactoring-in-simple-steps-III
+image:
+ src: /assets/img/custom/videos/complex-refactoring-in-simple-steps-III.jpg
+
+title: "Complex refactoring in simple steps Part III"
+hidden: true
+video-url: https://www.youtube.com/embed/SuqrAtXnn1U
+date: 2017-01-30 10:00:00 +00:00
+---
+
+
+## Screencast - Complex refactoring in simple steps, Part III From Conditional to Polymorphism
```

Comment:
This is internally consistent with codurance.com, but inconsistent with the videos. I suppose we could rename the three pages on our website, but that would break existing links.